server/src/main/kotlin/ir/armor/tachidesk/server/internal/About.kt

```diff
@@ -14,11 +14,9 @@ data class AboutDataClass(
     val revision: String,
 )
 
-object About {
-    fun getAbout(): AboutDataClass {
-        return AboutDataClass(
-            BuildConfig.version,
-            BuildConfig.revision,
-        )
-    }
```

Comment:
It's an object to keep the namespace clean, the package might get other objects/classes later
I follow this as a convention in all of the impl package

Reply:
duplicate